src/config/default-app-config.ts

```diff
@@ -297,6 +298,14 @@ export class DefaultAppConfig implements AppConfig {
     }
   };
 
+  suggestion: SuggestionConfig[] = [
+    // {
+    //   // Use this configuration to map a suggestion import to a specific collection based on the suggestion type.
```

Comment:
Can we describe what happens if this is *not* mapped?  Is there a default collection?

Also we may want to add a commented out example to `config.example.yml`.  That way it is documented in that example file as well.